packages/apps/spaces/app/auth/signin/signin.tsx

```diff
@@ -30,90 +24,82 @@ export default function SignIn({
 }) {
   return (
     <>
-      <GridItem h='100vh'>
-        <Box height='50%'>
+      <div className='h-screen'>
+        <div className='h-[50%]'>
           <Image
             alt=''
             src={BackgroundGridDot}
-            width={480}
-            top='-10%'
-            margin='auto'
+            className='top-[-10%] w-[480px]'
           />
-        </Box>
-        <Center height='100%' pos='relative' top='-50%'>
-          <Flex flexDirection={'column'} align={'center'} width={360}>
+        </div>
+        <div className='h-full flex items-center justify-center relative top-[-50%]'>
+          <div className='flex flex-col items-center w-[360px]'>
             <Image
               src={CustomOsLogo}
               alt='CustomerOS'
-              width={264}
-              height={264}
+              className='size-[264px]'
             />
-            <Heading color='gray.900' size='lg' py={3} mt={-10}>
+            <h2 className='text-gray-900 leading-9 font-bold text-3xl py-3 mt-[-40px]'>
               Welcome back
-            </Heading>
-            <Text color='gray.500'>Sign in to your account</Text>
+            </h2>
+            <p className='text-gray-500'>Sign in to your account</p>
             {providers &&
               Object.values(providers).map((provider, i) => {
                 let icon = undefined;
                 switch (provider.id) {
                   case 'google':
-                    icon = <Google boxSize={6} />;
+                    icon = <Google className='size-6' />;
                     break;
                   case 'azure-ad':
-                    icon = <Microsoft boxSize={6} />;
+                    icon = <Microsoft className='size-6' />;
                     break;
                   default:
                     icon = undefined;
                 }
+                const dynamicMargin = i === 0 ? 'mt-[17px]' : 'mt-3';
 
                 return (
                   <Button
-                    mt={i === 0 ? 6 : 3}
                     key={provider.name}
                     size='md'
                     variant='outline'
+                    colorScheme='gray'
                     leftIcon={icon}
-                    backgroundColor={'white'}
                     onClick={() => signIn(provider.id)}
-                    width='100%'
+                    className={cn(
+                      `w-[100%] py-[7px] px-[16px]  ${dynamicMargin} `,
```

Comment:
move `dynamicMargin` outside. Don't use interpolation for tw classes. also, when possible, use scale tokens like p-4 instead of p-[14px]